core/app/models/spree/variant.rb

```diff
@@ -100,8 +100,13 @@ class Variant < Spree::Base
       end
     end
 
-    self.whitelisted_ransackable_associations = %w[option_values product prices default_price]
-    self.whitelisted_ransackable_attributes = %w[weight sku]
+    self.whitelisted_ransackable_associations = %w(option_values prices default_price)
```

Comment:
```suggestion
    self.whitelisted_ransackable_associations = %w[option_values product prices default_price]
```

we need to keep `product` here